dex/README.md

```diff
@@ -1,31 +1,30 @@
-# serum-dex
+# Openbook DEX
 
 ## Deploying the DEX
 
-Using the `do.sh` script from the repository's top level directory,
+Run these commands from the repository's dex subdirectory directory,
```

Comment:
What do you think about just using

"Run these commands from this repository's `dex` directory"